install/install.sh

```diff
@@ -83,13 +83,18 @@ fetch() {
 }
 
 install() {
- local sudo 
+ local sudo
+ local msg
  if [ -w $BIN_DIR ]; then
-	 sudo=''
+	 sudo=""
+	 msg="Installing Starship, please wait…"
  else
-	 info "Root privileges are required to proceed the installation"
-	 sudo='sudo'
+	 warn "Escalated permission are required to install to ${BIN_DIR}"
+	 sudo -v || (error "Aborting installation(Pleave provide root password)";exit 1)
+	 sudo="sudo -v"
```

Comment:
I'm pretty sure this won't work and this line needs to be kept as `sudo='sudo'` because with `-v` it will "update user's timestamp **without running a command**" according to `--help`

Reply:
> I'm pretty sure this won't work

Do you mean the whole change?

Reply:
No only that line. Btw the error message appears to miss a space.

Reply:
I didn't get your point? is it only related to the `sudo -v` or the line that check if sudo password was dismissed?

Reply:
Your change sets the `sudo` variable that gets used for `${sudo} tar` to `sudo -v` in L94, but when `sudo` is used with `-v` flag, according to the docs the command will not get executed (here `tar`) and only the `sudo` validation check is performed (my local `sudo` actually drops me into usage help).

Reply:
Ok now I got it, and I noticed that about the `-v`